docs/src/app/components/pages/components/paper.jsx

```diff
@@ -64,7 +64,7 @@ const PaperPage = React.createClass ({
           },
           {
             name: 'zDepth',
-            type: 'number (0-5)',
+            type: 'oneOf ["0", "1", "2", "3", "4", "5"]',
```

Comment:
Good question, i just added quotes to every possibility in `oneOf`. could definitely change that for numbers. what do the rest think?